File: 2021/06/28/main.go
package main

func removeDuplicates(s string) string {
	l := len(s)
	t := []byte{}
	for i := 0; i < l; i++ {
		lt := len(t)
		if lt != 0 && s[i] == t[lt-1] {
			t = t[:lt-1]
		} else {
			t = append(t, s[i])
		}
	}
	return string(t)
}

File: 2021/06/28/main.py
class Solution:
    def removeDuplicates(self, s: str) -> str:
        l = []
        l.append(s[0])
        for i in range(1, len(s)):
            if len(l) != 0 and l[-1] == s[i]:
                l.pop()
            else:
                l.append(s[i])
        return ''.join(l)